.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Docker
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    build:
+        name: CI
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: Fix permissions
```

Comment:
Why do we need this step?

Reply:
I don't know, but it fails on github – https://github.com/wemake-services/kira-dependencies/actions/runs/3218053489/jobs/5261756464